.github/workflows/edgar-feed.yaml

```diff
@@ -0,0 +1,73 @@
+name: Rust CI
+
+on: [push, pull_request]
+
+defaults:
+  run:
+    working-directory: edgar-feed
+
+env:
+  CARGO_TERM_COLOR: always
+
+jobs:
+  linting:
+    name: linting with clippy
+    runs-on: ubuntu-latest
+    steps:
+      - name: Checkout code
+      - uses: actions/checkout@v2
+      - name: Install Rust toolchain
+      - uses: actions-rs/toolchain@v1
```

Comment:
Looks like some missing/mixed `name` and `uses` fields.

Reply:
fixed